src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/Rfc3161TimestampTokenInfo.cs

```diff
@@ -20,6 +23,20 @@ public sealed class Rfc3161TimestampTokenInfo
         private Oid? _hashAlgorithmId;
         private ReadOnlyMemory<byte>? _tsaNameBytes;
 
+        /// <summary>
+        /// Initializes a new instance of the Rfc3161TimestampTokenInfo class.
```

Comment:
You can use see cref to point to a real API and the intellisense xml will convert that to its full DocID:

```suggestion
        /// Initializes a new instance of the <see cref="Rfc3161TimestampTokenInfo" /> class with the specified parameters.
```